lib/BackgroundJob/RemoveEmptyRooms.php

```diff
@@ -77,13 +86,47 @@ protected function run($argument): void {
 	}
 
 	public function callback(Room $room): void {
-		if ($room->getType() === Room::CHANGELOG_CONVERSATION) {
+		$this->defineRoomToProcces($room);
+		$this->deleteIfIsEmpty();
+		$this->deleteIfFileIsRemoved();
+	}
+
+	private function defineRoomToProcces(Room $room) {
+		if ($room->getType() === Room::TYPE_CHANGELOG) {
+			return;
```

Comment:
if this is true, `deleteIfIsEmpty` and `deleteIfFileIsRemoved` will again be called with the old room object

Reply:
You found a :bug: 
I will write a test to cover this scenario.